simapp/app.go

```diff
@@ -4,26 +4,29 @@ import (
 	"io"
 	"os"
 
+	abci "github.com/tendermint/tendermint/abci/types"
```

Comment:
similar as my previous comment, should be start using `abcitypes` as the alias instead?
`dbm` vs `libsdb`, etc, etc.
These name changes add a lot of noise in my opinion.

Reply:
There are no import name changes in this block, I have simply just moved the block to follow the import order convention (see previous comment for order)